get-resource.sh

```diff
@@ -36,7 +36,18 @@ if [ -s "/shared/html/images/$RHCOS_IMAGE_FILENAME_OPENSTACK/$RHCOS_IMAGE_FILENA
     echo "$RHCOS_IMAGE_FILENAME_OPENSTACK/$RHCOS_IMAGE_FILENAME_COMPRESSED.md5sum found, contents:"
     cat /shared/html/images/$RHCOS_IMAGE_FILENAME_OPENSTACK/$RHCOS_IMAGE_FILENAME_COMPRESSED.md5sum
 else
-    curl -g --insecure --compressed -L -o "${RHCOS_IMAGE_FILENAME_RAW}" "${IMAGE_URL}/${RHCOS_IMAGE_FILENAME_RAW}"
+    CONNECT_TIMEOUT=120
+    MAX_ATTEMPTS=5
+
+    for i in $(seq ${MAX_ATTEMPTS}); do
```

Comment:
we could use until or while here and compress things a little bit, but this is already fine for me